cmd/main.go

```diff
@@ -23,7 +23,8 @@ import (
 // Variables set by goreleaser process: https://goreleaser.com/cookbooks/using-main.version.
 var (
 	// Version from git tag.
-	version = "dev"
+	version               = "dev"
+	defaultConfigLocation = "/alicenet/mainnet/config.toml"
```

Comment:
Default config location should probably default to the current directory.

Reply:
@vtleonardo what do you think?

Reply:
So, we are following the same standard as `geth`, by default the config files, chain data, etc are stored in the home directory (~/.ethereum). The `--init` command introduced by the PR #229 will create the config and setup folder on the $HOME/alicenet/mainnet, but the user will be able to choose a path. The same for `node` command, we are defaulting to the same folder of the `init` command, but the user can choose where he wants to read the config data from.

Reply:
Ah, right didn't notice that it was being joined. Works for me!